openshift/tests-extension/pkg/helpers/commands.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+)
+
+// findK8sClient returns "oc" if available, otherwise "kubectl".
+func findK8sClient() (string, error) {
+	clients := []string{"oc", "kubectl"}
+	for _, c := range clients {
+		if err := exec.Command(c, "version", "--client").Run(); err == nil {
+			return c, nil
+		}
+	}
+	return "", fmt.Errorf("no Kubernetes CLI client found (tried oc, kubectl)")
+}
+
+// RunK8sCommand runs a Kubernetes CLI command with the given args and returns combined output.
+func RunK8sCommand(ctx context.Context, args ...string) ([]byte, error) {
+	client, err := findK8sClient()
+	if err != nil {
+		return nil, err
+	}
+
+	cmd := exec.CommandContext(ctx, client, args...)
+	data, err := cmd.CombinedOutput()
```

Comment:
This could present problems, combining stdout and stderr. In case of an error (`err != nil`), the stderr is put into the error return.
[See: https://github.com/openshift/operator-framework-operator-controller/blob/2f8f54b35eb3cc626f9693bf23c0b475bdbdca54/openshift/tests-extension/pkg/env/cluster.go#L37](https://github.com/openshift/operator-framework-operator-controller/blob/2f8f54b35eb3cc626f9693bf23c0b475bdbdca54/openshift/tests-extension/test/olmv1-incompatible.go#L397-L404)

Reply:
Specifically: 
https://github.com/openshift/operator-framework-operator-controller/blob/2f8f54b35eb3cc626f9693bf23c0b475bdbdca54/openshift/tests-extension/test/olmv1-incompatible.go#L407-L416